src/cli/serve/serve.js

```diff
@@ -94,6 +95,14 @@ function applyConfigOverrides(rawConfig, opts, extraCliOptions) {
       set('server.ssl.certificate', DEV_SSL_CERT_PATH);
       set('server.ssl.key', DEV_SSL_KEY_PATH);
     }
+
+    if (opts.ssl && !opts.elasticsearch && !has('elasticsearch.hosts')) {
+      set('elasticsearch.hosts', 'https://localhost:9200');
```

Comment:
This doesn't seem to be the best way to hardcode the host. Maybe there's a better way to do this? Or to findout if Elasticsearch is running on a different port?

Note the certificate within `CA_CERT_PATH` is bound to localhost.

Reply:
If `--elasticsearch` or `--elasticsearch.hosts` is defined we should parse that url with `url.parse()`, throw if `parsedUrl.hostname !== 'localhost'`, and if it is then use `parsedUrl.port` and default to 9200.